lib/rialto/etl/transformers/people/names.rb

```diff
@@ -55,15 +55,15 @@ def fullname_from_names(given_name:, middle_name: nil, family_name:)
           # rubocop:disable Metrics/MethodLength
           def name_variations_from_names(given_name:, middle_name: nil, family_name:)
             name_variations = []
-            name_variations << "#{family_name}, #{given_name}"
-            name_variations << "#{given_name} #{family_name}"
-            if given_name
+            if given_name&.is_a?(String)
+              name_variations << "#{family_name}, #{given_name}"
+              name_variations << "#{given_name} #{family_name}"
               given_initial = given_name[0]
               name_variations << "#{family_name}, #{given_initial}"
               name_variations << "#{family_name}, #{given_initial}."
               name_variations << "#{given_initial} #{family_name}"
               name_variations << "#{given_initial}. #{family_name}"
-              if middle_name
+              if middle_name&.is_a?(String)
```

Comment:
Seems like this could be extracted into a method like `middle_name_variations`. What do you think?